### ISSUE TITLE
I probably should have asked you first

### ISSUE THREAD

**Issue by NJseo** [dudaster/eletheme#23]
I am trying to utilize custom fields and looper with a non-default, non-canvas, non-full width template with Elementor. I have tried making single and archive pages with it, but was not able to get elementor fields to connect. Bascially, I need the drop down menu for the posts to have the option of a secondary template, mytemplate.php to bring up the options for my custom fields ACF options and have the Elementor templates to apply to anything from that template.

Its been slow going trying to get that to work and I was hoping you would have suggestions.

**Comment by dudaster** [comment]
Have you tried Eletheme? "Looper" (ele-custom-skin) is compatible with it.
For custom fields just put the slug of the custom field inside double curly brackets: {{custom_field_slug}}.

You can even can add your own value:
```
// Add custom keywords to the eletheme
add_filter( 'eletheme_vars', 'my_keywords');
function my_keywords( $custom_vars ) {
    $custom_vars['current_year']=date('Y');
    return $custom_vars;
}
```
And you can use it in this case as {{current_year}} .

You can read more about it here: https://www.eletemplator.com/eletheme/

**Comment by NJseo** [comment]
Ok, I think I got it. I do have another question though.

So my search pages keep coming up blank and I think that is because I deleted a bunch of Elementor templates that came with one of your plugins or the Eletheme. I noticed when I opened the zip, that they were text files so I tried to reconstitute the _[default]_[search] template, but I still get a blank page. I am guessing these operate off of the id number and not the name of the template. So the question becomes how can I get them back in there without throwing away the other work that I have done on this project?
